src/python/WMCore/Services/AlertManager/AlertManagerAPI.py

```diff
@@ -0,0 +1,123 @@
+
+    """
+
+    configDict: the WMCore service configuration
+    alertName: a unique name for the alert
+    severity: low, medium, high
+    labels: a customizable dict of key:value pairs used for routing the alert, ex. "tag": "microservices"
+            must be configured in AlertManager by MONIT team
+    end: how many minutes until the alarm is silenced
+    generatorURL: this URL will be sent to AlertManager and configured as a clickable "Source" link in the web interface
+    annotations: a customizable dict of key:value pairs to be appended to the alert, typically:
+                 "hostname": "the hostname where the alert generated" (appended automatically)
+                 "summary": "a short description of the alert"
+                 "description": "a longer informational message with details about the alert"
+
+    AlertManager JSON format reference: https://www.prometheus.io/docs/alerting/latest/clients/
+    [
+      {
+        "labels": {
+        "alertname": "<requiredAlertName>",
+        "<labelname>": "<labelvalue>",
+        ...
+      },
+        "annotations": {
+          "<labelname>": "<labelvalue>",
+        ...
+      },
+        "startsAt": "<rfc3339>", # optional, will be current time if not present
+        "endsAt": "<rfc3339>",
+        "generatorURL": "<generator_url>"
+      },
+    ]
+    """
+
+    def __init__(self, configDict, alertName, severity, labels, end, generatorURL, annotations):
+        self.alertManagerUrl = configDict.get("alertManagerUrl")
+        self.alertName = alertName
+        self.end = end
+        self.severity = severity
+        self.labels = labels
+        self.generatorURL = generatorURL
+        self.annotations = annotations
+
+    def send(self):
+        headers = {"Content-Type": "application/json"}
+        request = []
+        alert = {}
+
+        self.labels["alertname"] = self.alertName
+        alert["labels"] = self.labels
+        # add the sender's hostname as an annotation
+        self.annotations["hostname"] = socket.gethostname()
+        alert["annotations"] = self.annotations
+
+        # In python3 we won't need the LocalTimezone class
+        # Will change to d = datetime.now().astimezone() + timedelta(minutes = self.end)
+        ltz = LocalTimezone()
+        d = datetime.now(ltz) + timedelta(minutes = self.end)
+        alert["endsAt"] = d.isoformat("T")
+        alert["generatorURL"] = self.generatorURL
+
+        request.append(alert)
+        # need to do this because pycurl_manager only accepts dict and encoded strings type
+        params = json.dumps(request)
+
+        mgr = RequestHandler()
+        res = mgr.getdata(self.alertManagerUrl, params=params, headers=headers, verb='POST')
+
+        return res
+
+# Below is a required python 2 class to determine current timezone
+# https://docs.python.org/2/library/datetime.html#tzinfo-objects
+# Everything below can be removed when we move to python3
+
+ZERO = timedelta(0)
```

Comment:
Erik, I have some general comments on this time/timezone handling. Can we move it under:
https://github.com/dmwm/WMCore/blob/master/src/python/Utils/Timers.py

and perhaps add these global variables inside an `__init__` method of the LocalTimeZone?

Given that it's a timezone information, it won't change during the process/service execution, so we could likely calculate it only once - in the `__init__` method of the `AlertManagerAPI` class and make that an object data (self.tz).